DatacenterIntegration/Portal/PortalUserDataGdprUtilities.psm1

```diff
@@ -61,7 +59,9 @@ function Initialize-UserDataClearEnv {
     if ($AutomationCredential) {
         $params.AutomationCredential = $AutomationCredential
     }
-    $refreshToken = Initialize-AzureRmUserRefreshToken @params
+    $azAccount = Initialize-AzAccount @param
```

Comment:
@rakku-ms Are you importing the identity module somewhere?

Reply:
yes @ line 41